src/dispatch/incident_type/models.py

```diff
@@ -15,7 +18,7 @@ class IncidentType(Base):
     slug = Column(String)
     description = Column(String)
     visibility = Column(String, default=Visibility.open)
-    plugin_metadata = Column(JSON, nullable=False, default=dict, server_default="{}")
+    plugin_metadata = Column(JSON, default=[])
```

Comment:
Shouldn't the default be an empty dict here?

Reply:
No, the new json format is a list of ducts.